src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Swift Package Manager for app developers
+description: How to use Swift Package Manager for native iOS or macOS dependencies
+---
+
+Flutter is migrating to [Swift Package Manager][] to manage iOS and macOS native
+dependencies.
+This is an experimental feature that might change in the future.
+It is currently only available on the [`main` channel][].
+Flutter will continue to support CocoaPods until further notice.
```

Comment:
Can we hoist this into an `:::important` banner at the top, or whatever strong indicator the rest of the site uses for experimental docs?

Reply:
I wrapped this in a `:::important` banner. @sfshaza2 Let me know if there's a better way to label experimental docs.

Reply:
All you need to do is:

:::important
Your content here
:::

Reply:
Done! :)